utility/src/main/java/org/oppia/android/util/parser/html/ImageTagHandler.kt

```diff
@@ -46,4 +48,23 @@ class ImageTagHandler(
       }
     } else consoleLogger.e("ImageTagHandler", "Failed to parse image tag")
   }
+
+  override fun handleContentDescription(
+    attributes: Attributes,
+    openIndex: Int,
+    closeIndex: Int,
+    output: Editable
+  ) {
+    val contentDescription = attributes.getJsonStringValue(CUSTOM_IMG_ALT_TEXT_ATTRIBUTE)
+    if (contentDescription != null) {
+      val spannableBuilder = SpannableStringBuilder(contentDescription)
+      spannableBuilder.setSpan(
+        contentDescription,
+        /* start= */ 0,
+        /* end= */ contentDescription.length,
+        Spannable.SPAN_INCLUSIVE_EXCLUSIVE
+      )
+      output.replace(openIndex, closeIndex, spannableBuilder)
+    } else consoleLogger.e("ImageTagHandler", "Failed to parse $CUSTOM_IMG_ALT_TEXT_ATTRIBUTE")
```

Comment:
Suggest making this a warning, instead, as the app still works but in a degraded state in this situation.